Source/Scene/Cesium3DTileset.js

```diff
@@ -1237,6 +1239,31 @@ define([
             get : function() {
                 return this._ellipsoid;
             }
+        },
+
+        /**
+         * Returns the <code>extras</code> property at the top-level of the tileset JSON, which contains application specific metadata.
+         * Returns <code>undefined</code> if <code>extras</code> does not exist.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @exception {DeveloperError} The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.
+         *
+         * @type {*}
+         * @readonly
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#specifying-extensions-and-application-specific-extras|Extras in the 3D Tiles specification.}
+         */
+        extras : {
+            get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this.ready) {
+                    throw new DeveloperError('The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.');
+                }
+                //>>includeEnd('debug');
```

Comment:
I forgot to mention, also add a test for this case. See `throws when getting asset and tileset is not ready`.

Reply:
Should be good now! https://github.com/AnalyticalGraphicsInc/cesium/pull/6974/commits/e4d20f171b05d13dd8a97e4c7c8defa1286ea969

Reply:
Thanks, I'll merge once CI passes.